opplast/upload.py

```diff
@@ -52,19 +54,26 @@ def upload(self, meta: dict) -> (bool, str):
 
         self.log.debug(f'Trying to upload "{video}" to YouTube...')
         path = str(Path.cwd() / video)
+
+
         self.driver.find_element_by_xpath(INPUT_FILE_VIDEO).send_keys(path)
         sleep(self.timeout)
 
+        modal = self.driver.find_element_by_css_selector(UPLOAD_DIALOG_MODAL)
+        self.log.debug('Found YouTube upload Dialog Modal')
+
         if title:
             if len(title) <= 100:
                 self.log.debug(f'Trying to set "{title}" as title...')
-                title_field = self.driver.find_element_by_id(TEXTBOX)
+                title_field = modal.find_element_by_id(TEXTBOX)
                 title_field.click()
                 sleep(self.timeout)
 
-                title_field.clear()
-                sleep(self.timeout)
+                # clearing out title which defaults to filename
+                for i in range(30):

```

Comment:
Wouldn't this be better?
```python
 for i in range(len(title_field.text)): 
``` 
If the file name is longer than 30 characters, it wouldn't get completely cleared.

Reply:
i added a sleep delay and +10 because sometime a backspace might be missing :) just in case!